src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -111,18 +145,21 @@ def __call__(self,
             metadata = _IMMUTABLE_EMPTY_TUPLE
 
         if not self._interceptors:
-            return UnaryUnaryCall(request, _timeout_to_deadline(timeout),
+            call = UnaryUnaryCall(request, _timeout_to_deadline(timeout),
```

Comment:
I'm somewhat worried about reference cycles here. Would it be feasible to keep a collection of weakrefs instead?

Reply:
Yeps changed for using `WeakSet`